### PR TITLE
create models

### PR DESCRIPTION
**Pull request by louy755**

### CODE REVIEW

File: app/models/client.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class Client < ApplicationRecord
+  has_many :client_files
+
+  def initials
```
Comment (ssharba) [inline app/models/client.rb]: what is this method for?

File: app/models/client_file.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+class ClientFile < ApplicationRecord
+  belongs_to :client
+
+  before_save { save_regex }
```
Comment (ssharba) [inline app/models/client_file.rb]: there is nothing in the task requirements for this part.

**Review by ssharba — CHANGES_REQUESTED** [review]

File: app/models/client.rb (hunk 1 above)
Comment (louy755) [reply app/models/client.rb]: This method to slice or split the name from client so we can save it like this 
(initial_my_file_11092017.txt)

File: app/models/client_file.rb (hunk 2 above)
Comment (louy755) [reply app/models/client_file.rb]: (my_file_11092017.txt)
this part as you wrote but I am not sure but we can change it!
also I have this result from my terminal as the following format : 
ALI_moon_2017-12-05.txt

Comment (louy755) [reply app/models/client_file.rb]: but I can clear the models and start with your instructions!

**Review by ssharba — APPROVED** [review]

**Pull request merged**